src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -373,10 +373,12 @@ public static bool Exists([NotNull]string name, [CanBeNull]string directory)
 		[Obsolete("This has been superseded by Database.Log.Console.  This function is now a no-op")]
         [ExcludeFromCodeCoverage]
 		public static void SetLogLevel(LogDomain domains, LogLevel level)
-		{
-		    WriteLog.To.Database.W(Tag, "Database.SetLogLevel is deprecated, use Database.Log.Console instead.");
-		}
-        
+        {
+            Log.Console.Level = level;
+            Log.Console.Domains = domains;
+            WriteLog.To.Database.W(Tag, "Currently it will only affect the console logger and not the file logger.");
```

Comment:
Should we make this error message cross platform? Does this apply to all platform or just .Net?

Reply:
This message is .NET specific.  The others never had this API affect the file logger (it didn't exist for them until the new log API was created)